bot/exts/evergreen/bookmark.py

```diff
@@ -1,21 +1,88 @@
+import asyncio
 import logging
 import random
 
 import discord
 from discord.ext import commands
 
-from bot.constants import Colours, ERROR_REPLIES, Emojis, Icons
+from bot.constants import Colours, ERROR_REPLIES, Icons
 from bot.utils.converters import WrappedMessageConverter
 
 log = logging.getLogger(__name__)
 
+# Number of seconds to wait for other users to bookmark the same message
+TIMEOUT = 120
+BOOKMARK_EMOJI = "📌"
+
 
 class Bookmark(commands.Cog):
     """Creates personal bookmarks by relaying a message link to the user's DMs."""
 
     def __init__(self, bot: commands.Bot):
```

Comment:
In #718, all `commands.Bot` type hints were changed to `Bot` from `bot/bot.py`; we should probably use that here (and in the setup function L143) too to stay consistent